internal/tool/tool.go

```diff
@@ -137,6 +158,20 @@ func commandParameters(configFile *os.File, filesToAnalyse []string) []string {
 	return cmdParams
 }
 
+// TODO change this func
+func getFileLanguageFromExtension(filesToAnalyse []string) []string {
```

Comment:
we should aim to use the same extensions as defined here:
https://github.com/codacy/codacy-plugins-api/blob/e94cfa10a5f2eafdeeeb91e30a39e2032e1e4cc7/codacy-plugins-api/src/main/scala/com/codacy/plugins/api/languages/Language.scala#L100